datasetinsights/datasets/protos/string_int_label_map_pb2.py

```diff
@@ -0,0 +1,185 @@
+# -*- coding: utf-8 -*-
```

Comment:
Why is this file modified? This is a file that was generated by protobuf and it should never be modified manually.

Reply:
Its been modified by black. linting fix. After restructuring its assuming it a new file.

$ black .
reformatted /Users/sanjay.vishwakarma/Desktop/projects/thea_dir_change/datasetinsights/datasets/protos/string_int_label_map_pb2.py